docs/topics/about-vscode-extension.adoc

```diff
@@ -4,9 +4,9 @@
 
 
 [id='about-vscode-extension_{context}']
-= About the {ProductName} extension for Visual Studio Code
+= About the {ProductShortName} extension for Visual Studio Code
 
-The {ProductName} ({ProductShortName}) extension provides assistance directly in Visual Studio Code for developers migrating or modernizing projects that contain Java applications.
+The {ProductName} ({ProductShortName}) extension provides assistance directly in Visual Studio Code for developers migrating or modernizing projects that contain Java applications. The {ProductShortName} extension is compatible with Visual Studio Codespaces, Microsoft's cloud-hosted development environment.
```

Comment:
IBM style guide [discourages use of possessive form](https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives) for product names, so I think that would also apply to `Microsoft's`.